### PR TITLE
[IMP] web: convert mobile many2many_tags_field test to hoot

### PR DESCRIPTION
**Pull request by res-odoo**
task-4028335

Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/tests/views/fields/many2many_tags_field.test.js [hunk 1]
```diff
@@ -1527,23 +1839,23 @@ test("Many2ManyTagsField selected records still pickable and not duplicable", as
             </list>
         `,
     });
-
+    const item = getMockEnv().isSmall ? ".o_kanban_record" : "a.dropdown-item";
```
Comment (pparidans) [inline addons/web/static/tests/views/fields/many2many_tags_field.test.js:1842]: I think this is a good candidate for having both desktop and mobile version of this test

File: addons/web/static/tests/views/fields/many2many_tags_field.test.js [hunk 2]
```diff
@@ -1316,7 +1517,10 @@ test("set a required many2many_tags and save directly", async () => {
         message: "The tag is displayed, but the web read is not finished yet",
     });
 
-    expect.verifySteps(["name_search", "web_read"]);
+    const steps = getMockEnv().isSmall
```
Comment (pparidans) [inline addons/web/static/tests/views/fields/many2many_tags_field.test.js:1520]: I think this is a candidate for having both desktop and mobile version of this test...
But do we actually have a compelling reason to check those steps in this test's context ? :thinking:

Comment (adr-odoo) [reply addons/web/static/tests/views/fields/many2many_tags_field.test.js]: assertion removed

**Review by adr-odoo — APPROVED** [review]
@robodoo rebase-ff r+

**Pull request closed without merging**